recipes/qpid-proton/all/conanfile.py

```diff
@@ -0,0 +1,144 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.microsoft import is_msvc
+
+import os
+
+required_conan_version = ">=2.1.0"
+
+class QpidProtonConan(ConanFile):
+    version = "0.40.0"
+    name = "qpid-proton"
+    description = "Qpid Proton is a high-performance, lightweight messaging library."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://qpid.apache.org/proton/"
+    topics = ("conan", "qpid", "proton", "messaging", "message", "queue", "topic", "mq", "amqp")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "tools": [True, False],
+        "with_opentelemetry": [True, False],
```

Comment:
```suggestion
```

Let's keep open telemetry as permanent dependency. Unless there is a reason for not using it.